diff --git a/node_modules/@yourdash/uikit/components/image/image.tsx b/node_modules/@yourdash/uikit/components/image/image.tsx
index 5195ce3..a8e17be 100644
--- a/node_modules/@yourdash/uikit/components/image/image.tsx
+++ b/node_modules/@yourdash/uikit/components/image/image.tsx
@@ -11,13 +11,12 @@ const Image: FC<{
   src: string;
   accessibleLabel: string;
   className?: string;
-  authenticatedImage?: boolean;
   disableLazyLoading?: boolean;
 }> = (props) => {
   return (
     <img
       className={`${styles.component} ${props.className}`}
-      src={(props.authenticatedImage ? csi.getInstanceUrl() : "") + props.src}
+      src={props.src}
       draggable={false}
       loading={props.disableLazyLoading ? "eager" : "lazy"}
       alt={props.accessibleLabel}
