diff --git a/node_modules/@yourdash/uikit/components/image/image.tsx b/node_modules/@yourdash/uikit/components/image/image.tsx
index 5672533..71e7d86 100644
--- a/node_modules/@yourdash/uikit/components/image/image.tsx
+++ b/node_modules/@yourdash/uikit/components/image/image.tsx
@@ -3,7 +3,7 @@
  * YourDash is licensed under the MIT License. (https://ewsgit.mit-license.org)
  */
 
-import coreCSI from "@yourdash/csi/coreCSI.ts";
+// import coreCSI from "@yourdash/csi/coreCSI.ts";
 import Icon from "../icon/icon.tsx";
 import { UKIcon } from "../icon/iconDictionary.ts";
 import styles from "./image.module.scss";
@@ -15,7 +15,7 @@ const Image: FC<{
   accessibleLabel: string;
   containerClassName?: string;
   className?: string;
-  authenticatedImage?: boolean;
+  // authenticatedImage?: boolean;
   disableLazyLoading?: boolean;
   noRounding?: boolean;
 }> = (props) => {
@@ -69,7 +69,7 @@ const Image: FC<{
           onLoad={(e) => {
             setLoaded(e.currentTarget.complete);
           }}
-          src={(props.authenticatedImage ? coreCSI.getInstanceUrl() : "") + src}
+          src={/*(props.authenticatedImage ? coreCSI.getInstanceUrl() : "") +*/ src}
         />
       ) : (
         <Icon icon={UKIcon.ServerError} />
