diff --git a/node_modules/@yourdash/uikit/components/image/image.tsx b/node_modules/@yourdash/uikit/components/image/image.tsx
index 5195ce3..de7b01e 100644
--- a/node_modules/@yourdash/uikit/components/image/image.tsx
+++ b/node_modules/@yourdash/uikit/components/image/image.tsx
@@ -3,7 +3,6 @@
  * YourDash is licensed under the MIT License. (https://ewsgit.mit-license.org)
  */
 
-import csi from "@yourdash/csi/csi.js";
 import styles from "./image.module.scss";
 import { FC } from "react";
 
@@ -11,13 +10,12 @@ const Image: FC<{
   src: string;
   accessibleLabel: string;
   className?: string;
-  authenticatedImage?: boolean;
   disableLazyLoading?: boolean;
 }> = (props) => {
   return (
     <img
       className={`${styles.component} ${props.className}`}
-      src={(props.authenticatedImage ? csi.getInstanceUrl() : "") + props.src}
+      src={props.src}
       draggable={false}
       loading={props.disableLazyLoading ? "eager" : "lazy"}
       alt={props.accessibleLabel}
